src/constants/feature-lists-data.ts

```diff
@@ -98,7 +98,7 @@ const featureListData = {
         items: [
           {
             content:
-              'Add auth to apps using the Amplify CLI and Studio. Support for login, MFA, social providers, and CAPTCHA protection. Integrate auth with intuitive client library APIs.',
+              'Easily configure auth for your app with Amplify CLI and Studio—supports login, MFA, social providers, and more. Then integrate auth with intuitive client library APIs.',
             linkText: 'Simple Configuration',
```

Comment:
This should be "Simple configuration"

Note the lowercase 'c'

Reply:
Done